EIPS/eip-4974.md

```diff
@@ -12,9 +12,9 @@ requires: 165
 ---
 
 ## Abstract
-The following describes a standard interface for fungible non-tradable tokens, or EXP. This standard provides basic functionality for participant addresses to consent to receive tokens and for an operator address to transfer tokens.
+The following describes a standard interface for assignment of numerical ratings. This standard provides basic functionality for an operator address to assign and update ratings.
 
-EXP, shorthand for "experience points", may represent accumulated recognition within a smart contract. Like experience points in video games, citations on an academic paper, or Reddit Karma, EXP is bestowed for useful contributions, accumulates as indistinguishable units, and should only be reallocated or destroyed by a reliable authority so empowered.
+Ratings may represent accumulated recognition within a smart contract. Like experience points in video games, citations on an academic paper, or Reddit Karma, EXP is bestowed for useful contributions, accumulates as indistinguishable units, and should only be reallocated or destroyed by a reliable authority so empowered.
```

Comment:
You are removing the blurb about what exp point are and not including an explanation in the new line.

Reply:
Thanks for the note. Fully updated now.